docs-v2/content/en/samples/verify/verify.yaml

```diff
@@ -12,13 +12,15 @@ verify:
     name: integration-test-container
     image: integration-test-container
   executionMode:
-    kubernetesCluster: {}
+    kubernetesCluster:
```

Comment:
Can this be commented out w/ some indication the field is optional?

Reply:
Good idea. I made a couple changes to help clarify that these fields are optional:
1. Changed wording on line 28 to "There are two ways to optionally customize the Skaffold-generated Kubernetes Job"
2. Added "optional" to describe both override configuration options on lines 37 and 38
3. Commented out the optional lines in the YAML and added "[optional]" above to further clarify.
Let me know what you think!